src/fides/api/alembic/migrations/versions/4ced99dabebb_add_location_regulation_selections_table.py

```diff
@@ -0,0 +1,59 @@
+"""add location regulation selections table
+
+Revision ID: 4ced99dabebb
+Revises: 548a1ac26771
+Create Date: 2023-12-18 13:50:50.620939
+
+"""
+import sqlalchemy as sa
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "4ced99dabebb"
+down_revision = "548a1ac26771"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.create_table(
+        "location_regulation_selections",
+        sa.Column("id", sa.String(length=255), nullable=False),
+        sa.Column(
+            "created_at",
+            sa.DateTime(timezone=True),
+            server_default=sa.text("now()"),
+            nullable=True,
+        ),
+        sa.Column(
+            "updated_at",
+            sa.DateTime(timezone=True),
+            server_default=sa.text("now()"),
+            nullable=True,
+        ),
+        sa.Column("selected_locations", sa.ARRAY(sa.String()), nullable=False),
+        sa.Column("selected_regulations", sa.ARRAY(sa.String()), nullable=False),
+        sa.Column(
+            "single_row",
+            sa.Boolean(),
+            nullable=False,
+        ),
+        sa.PrimaryKeyConstraint("id"),
+        sa.CheckConstraint("single_row", name="single_row_check"),
```

Comment:
I didn't know it was possible to have a single row constraint. That's cool!

Reply:
ahh, I see. Only one row can have the true boolean value and that value is default to `true`

Reply:
haha yup, it's a bit hacky but i think this is the best way to do it!